src/Data/List/Relation/Unary/Any/Properties.agda

```diff
@@ -97,6 +97,11 @@ Any-cong {P = P} {Q = Q} {xs = xs} {ys} P↔Q xs≈ys =
 ------------------------------------------------------------------------
 -- Any.map
 
+map-ext : (f g : P ⋐ P) → (∀ {x} (p : P x) → f p ≡ g p) →
+          (p : Any P xs) → Any.map f p ≡ Any.map g p
+map-ext f g hyp (here  p) = cong here (hyp p)
+map-ext f g hyp (there p) = cong there $ map-ext f g hyp p
```

Comment:
Missing in CHANGELOG?

Reply:
Yes. I think that in fact this lemma, which never gets used, should in fact be removed... I mistakenly believed I could use to refactor `map-id` and `map-∘`...

Reply:
Aaaargh! Just realised that we also have (legacy again!):
* `Data.List.Relation.Unary.Any.Properties`
```agda
map-id : ∀ (f : P ⋐ P) → (∀ {x} (p : P x) → f p ≡ p) →
         (p : Any P xs) → Any.map f p ≡ p

map-∘ : ∀ (f : Q ⋐ R) (g : P ⋐ Q) (p : Any P xs) →
        Any.map (f ∘ g) p ≡ Any.map f (Any.map g p)
```
and
* `Data.List.Relation.Unary.All.Properties`
```agda
map-cong : ∀ {f : P ⋐ Q} {g : P ⋐ Q} (pxs : All P xs) →
           (∀ {x} → f {x} ≗ g) → All.map f pxs ≡ All.map g pxs
map-cong []         _   = refl
map-cong (px ∷ pxs) feq = cong₂ _∷_ (feq px) (map-cong pxs feq)

map-compose : ∀ {f : P ⋐ Q} {g : Q ⋐ R} (pxs : All P xs) →
              All.map g (All.map f pxs) ≡ All.map (g ∘ f) pxs
map-compose []         = refl
map-compose (px ∷ pxs) = cong (_ ∷_) (map-compose pxs)
```

So: should we
* keep the new lemma, rename it to `map-cong`, and generalise it as suggested?
* rename `map-compose` in `All.properties` to `map-∘` and deprecate the former?

Ugh: PR mission-creep!

Reply:
DONE. Pushed.